Core/Core.Platform.Android/MobileCoreAndroid.cs

```diff
@@ -46,10 +51,19 @@ public static void Init(Assembly assembly, Context appContext)
         /// <param name="appContext">Android application contextt</param>
         /// <param name="options">additional initialization options</param>
         public static void Init(Assembly assembly, Context appContext, Options options)
-        {
+        {
+            // TODO: check if already initialized
+            RegisterServices();
             IPlatformInjector platformInjector = new AndroidPlatformInjector(appContext);
             platformInjector.ExecutingAssembly = assembly;
             MobileCore.Init(platformInjector, options);
         }
+
+        private static void RegisterServices()
+        {
+            ServiceFinder.RegisterType<IAuthService, AuthService>();
```

Comment:
as I commented above, the problem here is that the android implementation of `AuthService` may not exist in user's project.